tracer/test/test-applications/security/Samples.Security.AspNetCore5/IdentityStores/RoleStore.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace Samples.Security.AspNetCore5.IdentityStores;
+
+public class RoleStore : IRoleStore<IdentityRole>
```

Comment:
Ah, implementing custom `IRoleStore<>` this takes me back 😂 

Any reason you couldn't use the built-in EF Core SQLite stores? 🤔

Reply:
Yeah, fair enough! Though I wonder if we should have a single "smoke test" for one framework using EF core, as that's how 99% of people will use it? 🤔 Probably not necessary, and doesn't need to be this PR, but might be a good idea to be sure 🙂

Reply:
makes sense yes thanks, I'll create a backlog, would this require creating a new sample? maybe we could also have EF on one framework only in the security aspnet core 5 sample nd one test for this framework only too

Reply:
In the end I added EF Core to Net 7 under Samples.Security.AspNetCore5 and other versions with the ado sql lite provider and it seems to be working 🎆

Reply:
Also ended up adding instrumentation of the succesful login in HttpContextExtensions in 917cd737cfedf6425993d0698d3559446cd47b1f